docs/_includes/header-title.adoc

```diff
@@ -5,7 +5,7 @@ Included in:
 ////
 
 A document title must be placed on the first line of the document.
-It is preceded by one equal sign followed by at least one space (`=`), and then the text of the title.
+It is preceded by one equal sign followed by at least one space (`=&nbsp;`), and then the text of the title.
```

Comment:
To enter a non-breaking space, I would prefer that you use `{nbsp}`. This ensures that we aren't hard coding HTML entities.

Technically, you could just use a literal space, but that relies on the CSS to declare `whitespace: nowrap` for inline `<code>` elements, so perhaps `{nbsp}` is the right choice here.